pkgs/tools/networking/mosh/default.nix

```diff
@@ -1,5 +1,6 @@
-{ stdenv, fetchurl, zlib, protobuf, ncurses, pkgconfig, IOTty
-, makeWrapper, perl, openssl, autoreconfHook, openssh, bash-completion }:
+{ lib, stdenv, fetchurl, zlib, protobuf, ncurses, pkgconfig, IOTty
+, makeWrapper, perl, openssl, autoreconfHook, openssh, bash-completion
+, libutempter, withUtempter ? true }:
```

Comment:
`libutempter ? null, withUtempter ? stdenv.isLinux` should fix mosh on darwin

Reply:
thanks :) let's see if the borg builds correctly and then I'll squash this for merging :)

Reply:
you don't need to squash, I can do that when merging